pmm-app/src/shared/components/helpers/validators.ts

```diff
@@ -29,6 +29,14 @@ export const validators = {
     return value >= from && value <= to ? undefined : `Value should be in the range from ${from} to ${to}`;
   },
 
+  min: (from) => (value) => {
+    if (!value) {
```

Comment:
`value = 0` seems to qualify for a min value but would fail in such case.

Reply:
Fixed!!